src/e2e/java/teammates/e2e/pageobjects/InstructorHomePage.java

```diff
@@ -119,14 +119,14 @@ public void deleteSession(int courseTabIndex, int sessionIndex) {
     public void archiveCourse(int courseTabIndex) {
         WebElement courseTab = getCourseTab(courseTabIndex);
         click(courseTab.findElement(By.className("btn-course")));
-        clickAndConfirm(courseTab.findElement(By.className("btn-archive-course")));
+        clickAndConfirm(browser.driver.findElement(By.cssSelector("body > div > div >a.btn-archive-course.btn")));
```

Comment:
Can this be shortened to `body > div > div > .btn-archive-course`? Seems a bit more readable to me, and conveys the same meaning.

Reply:
No & I have mentioned the reason for it too below 
you can see it in [instructor-home-page.component.spec.ts] for archive button